apcd-cms/src/apps/view_users/static/view_users_table/css/table.css

```diff
@@ -0,0 +1,129 @@
+.users-table {
+    --cell-padding: 5px;
+
+    width: 100%;
+    table-layout: fixed; /* allows table to shrink (reason unknown) */
```

Comment:
```suggestion
    table-layout: inherit; /* allows table to shrink (reason unknown) */   
```

Reply:
Added these changes so email address don't run over onto the contents of the name column